lib/validators/doc-container-groups-validator.ts

```diff
@@ -26,7 +26,7 @@ export class DocContainerGroupsValidator extends Validator {
      * @param errorReporter
      * @param parsedComponent
      */
-    validateComponent(parsedComponent: ParsedComponent): void {
+    validateComponent(parsedComponent: Component): void {
```

Comment:
There are multiple instances where the variable is still called `parsedComponent`. I guess we want to rename those as well?

Reply:
Done 👍